src/main/java/frc/team2412/robot/commands/intake/IntakeDefaultCommand.java

```diff
@@ -6,12 +6,12 @@ public class IntakeDefaultCommand extends IntakeSetCommand {
 
 	public IntakeDefaultCommand(IntakeSubsystem intakeSubsystem) {
 
-		super(intakeSubsystem, 0);
+		super(intakeSubsystem, 0.1);
```

Comment:
I'm guessing this was tested but note that this set to intake speed is only done at the start of each robot code cycle (during robotInit).